scripts/manageacl.py

```diff
@@ -75,53 +78,75 @@ async def run(self):
                 await self.remove_all_acls(search_client)
             elif self.acl_action == "add":
                 await self.add_acl(search_client)
+            elif self.acl_action == "update_storage_urls":
```

Comment:
Is this doc'd in the readme?

Reply:
I documented it in deploy_features: https://github.com/Azure-Samples/azure-search-openai-demo/pull/1535/files#diff-75e3da8fd5af9aaa591ace5f5653b494b44dbc24366b287e735f675ee8f0c14bR145
Will also put it in the release notes.